src/cljs/metafacture_playground/views.cljs

```diff
@@ -41,27 +42,36 @@
 (def input (component "Input"))
 (def popup (component "Popup"))
 
+;;; Using monaco editor react component
+
+(def monaco-editor (g/get monaco-react "default"))
+
 ;;; Color and Theming
 
 (def color "blue")
 (def basic-buttons? true)
 
-; Sizes of input fields
+; Config of input fields
+
+(def focused-editor "data")
 
 (def data-config
   {:name "data"
    :width 16
-   :rows 5})
+   :height "150px"
+   :language "text/plain"})
 
 (def fix-config
   {:name "fix"
    :width 8
-   :rows 15})
+   :height "250px"
+   :language "text/plain"})
```

Comment:
Would it be possible to register a custom `fix` language and use some of the config files from the extension, in particular [fix.configuration.json](https://github.com/metafacture/metafacture-fix/blob/master/org.metafacture.fix.vsc/fix.configuration.json), [fix.snippets.json](https://github.com/metafacture/metafacture-fix/blob/master/org.metafacture.fix.vsc/fix.snippets.json), and [fix.tmLanguage](https://github.com/metafacture/metafacture-fix/blob/master/org.metafacture.fix.vsc/fix.tmLanguage), here in the widget?

Reply:
It's possible to register a fix language (and that's the plan) but if I'm informed right it's not possible to directly use these files. The rules can be reused but must be converted to something like [this](https://microsoft.github.io/monaco-editor/playground.html#extending-language-services-custom-languages). I thought that would rather be part of a ticket like #4. If you think that belongs to this ticket I will move it to working again.

Reply:
> I thought that would rather be part of a ticket like #4.

Right, from the title that would fit. The original idea for #4 was to be based on the language server. I think it's worth to explore how far we get based on the config files alone before adding LSP features. We can do that here or in #4, whatever fits better for you.

Reply:
How about a new ticket fot this feature? (Discussed in more detail in the next grooming?)
With a new ticket we get the the use of the 'text/plain'-Monaco Editors faster into main and we don't mix it with adding LSP features.

Reply:
Yes, new ticket sounds good.